tests/acceptance/features/apiSpaces/changeSpaces.feature

```diff
@@ -86,58 +90,26 @@ Feature: Change data of space
 
   Scenario Outline: viewer and editor cannot change the description(subtitle) of a space via the Graph API
     When user "<user>" changes the description of the "Project Jupiter" space to "The Death Star is a fictional mobile space station"
-    Then the HTTP status code should be "<code>"
+    Then the HTTP status code should be "403"
     Examples:
-      | user  | code |
-      | Brian | 403  |
-      | Bob   | 403  |
+      | user  |
+      | Brian |
+      | Bob   |
 
 
-  Scenario Outline: user tries to increase the quota of a Space via the Graph API
+  Scenario Outline: user with normal space permission can't increases the quota of a Space via the Graph API
     When user "<user>" changes the quota of the "Project Jupiter" space to "100"
-    Then the HTTP status code should be "<code>"
-    And for user "<user>" the JSON response should contain space called "Project Jupiter" and match
-
-    """
-     {
-      "type": "object",
-      "required": [
-        "name",
-        "quota"
-      ],
-      "properties": {
-        "name": {
-          "type": "string",
-          "enum": ["Project Jupiter"]
-        },
-        "quota": {
-          "type": "object",
-          "required": [
-            "total"
-          ],
-          "properties": {
-            "total" : {
-              "type": "number",
-              "enum": [<expectedQuotaValue>]
-            }
-          }
-        }
-      }
-    }
-    """
+    Then the HTTP status code should be "403"
     Examples:
-      | user  | code | expectedQuotaValue |
-      | Alice | 200  | 100                |
-      | Brian | 403  | 20                 |
-      | Bob   | 403  | 20                 |
+      | user  |
+      | Brian |
+      | Bob   |
 
 
   Scenario Outline: space admin user set no restriction quota of a Space via the Graph API
     When user "Alice" changes the quota of the "Project Jupiter" space to "<quotaValue>"
     Then the HTTP status code should be "200"
-    When user "Alice" uploads a file inside space "Project Jupiter" with content "some content" to "file.txt" using the WebDAV API
-    Then the HTTP status code should be "201"
-    And for user "Alice" the JSON response should contain space called "Project Jupiter" and match
```

Comment:
Maybe we can transfer this scenario to a feature file related to quota if there's not already one there?  I see other scenarios related to quota in this feature file as well should they be here. Also, this doesn't need to be done in this PR. It's just that I noticed it here so I'm mentioning it. 

cc @SagarGi @saw-jan do we need an issue with this refactoring? or it's okay to have this scenario in this file?

Reply:
Should carefully organize scenarios. but in later PRs

Reply:
Yes in later PRs. I'm just saying should we create a separate issue so we don't forget?

Reply:
Yeah, if there is none then we should open an issue to work on it later